rslib/src/deckconfig/schema11.rs

```diff
@@ -51,6 +51,7 @@ pub struct DeckConfSchema11 {
     // NOTE: if adding new ones, make sure to update clear_other_duplicates()
     #[serde(default)]
     new_mix: i32,
+    /// This has been repurposed as a boolean: `new_ignore_review_limit`
```

Comment:
Since this field was never used, another option would be to just drop the existing key and create a new one that's a bool.